FMT.ipynb

```diff
@@ -0,0 +1,908 @@
+      "--2020-03-18 10:55:42--  https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/fmt/fmt-tutorial-demux-1-1p.qza\n",
+      "Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 52.218.250.32\n",
+      "Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|52.218.250.32|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 2165499 (2.1M) [binary/octet-stream]\n",
+      "Saving to: ‘fmt-tutorial-demux-1.qza’\n",
+      "\n",
+      "fmt-tutorial-demux- 100%[===================>]   2.06M   474KB/s    in 4.6s    \n",
+      "\n",
+      "2020-03-18 10:55:48 (458 KB/s) - ‘fmt-tutorial-demux-1.qza’ saved [2165499/2165499]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -O $workdir/\"fmt-tutorial-demux-1.qza\" \\\n",
+    "  \"https://data.qiime2.org/2020.2/tutorials/fmt/fmt-tutorial-demux-1-1p.qza\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2020-04-15 09:31:04--  https://data.qiime2.org/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza\n",
+      "Resolving data.qiime2.org (data.qiime2.org)... 52.35.38.247\n",
+      "Connecting to data.qiime2.org (data.qiime2.org)|52.35.38.247|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 FOUND\n",
+      "Location: https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza [following]\n",
+      "--2020-04-15 09:31:05--  https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza\n",
+      "Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 52.218.216.56\n",
+      "Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|52.218.216.56|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 935392 (913K) [binary/octet-stream]\n",
+      "Saving to: ‘/home/user/qiime2-moving-pictures-tutorial//fmt-tutorial-demux-2.qza’\n",
+      "\n",
+      "/home/user/qiime2-m 100%[===================>] 913.47K   490KB/s    in 1.9s    \n",
+      "\n",
+      "2020-04-15 09:31:07 (490 KB/s) - ‘/home/user/qiime2-moving-pictures-tutorial//fmt-tutorial-demux-2.qza’ saved [935392/935392]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -O $workdir/\"fmt-tutorial-demux-2.qza\" \\\n",
+    "  \"https://data.qiime2.org/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Importing data as a qiime2 artifact\n",
+    "\n",
+    "All data that is used as input to QIIME 2 is in form of QIIME 2 artifacts, which contain information about the type of data and the source of the data. The first thing we need to do is import these sequence data files into a QIIME 2 artifact.\n",
```

Comment:
It should be clear than in the next section the user should consciously choose one of the options: 10 or 1 % subsample data before importing the artifacts.